main.js

```diff
@@ -59,29 +61,54 @@ function displayGrid() {
         <p>${ideas[i].title}</p>
         <p>${ideas[i].body}</p>
       </div>
-      <footer class="card-footer">
-        <img src="./assets/comment.svg" alt="Comment">Comment
-      </footer>
+      <div>
+        <footer class="card-footer">
+          <img id="add-comment" src="./assets/comment.svg" alt="Comment">Comment
+        </footer>
+        <div id="${i}" class="hidden">
```

Comment:
I have uneasy feelings about this method of generating ids, but thus far it's causing no problems, so *shrug*